shadows/framework/src/main/java/org/robolectric/shadows/ShadowLocaleManager.java

```diff
@@ -30,6 +33,7 @@ public class ShadowLocaleManager {
    * @see #enforceInstallerCheck
    * @see #setCallerAsInstallerForPackage
    */
+  @RequiresApi(api = VERSION_CODES.N)
```

Comment:
Why `N`? LocaleManager is T and above?

Reply:
Yep. This file only supports from T, and N is generated by Android Studio suggestion as this method uses API added from N.